src/extensions/dead-clicks-autocapture.ts

```diff
@@ -36,9 +36,14 @@ export class DeadClicksAutocapture {
         this.startIfEnabled()
     }
 
-    public startIfEnabled() {
-        if (this.isEnabled) {
-            this.loadScript(this.start.bind(this))
+    public startIfEnabled({
+        force,
+        onCapture,
+    }: { force?: boolean; onCapture?: (click: DeadClickCandidate, properties: Properties) => void } = {}) {
+        if (this.isEnabled || force) {
+            this.loadScript(() => {
+                this.start(onCapture)
+            })
         }
     }
```

Comment:
not totally convinced by this signature change

passing in `onCapture` allows only one capture per instance and is at least explicit
passing in `force` to enable the capture feels messy... you could pass in the `isEnabled` method instead 🤷